### ISSUE TITLE
download the dict error

### ISSUE THREAD

**Issue by k3mlol** [OWASP/Amass#266]
I just found the error below
**version:v3.1.4**
I download from the release page
https://github.com/OWASP/Amass/releases/download/v3.1.4/amass_v3.1.4_linux_amd64.zip

``` bash
./amass enum -d baidu.com
```
``` html
Improper mask used: <a class="btn btn-sm btn-with-count tooltipped tooltipped-s" aria-label="you must be signed in to star a repository" rel="nofollow" data-hydro-click="{&quot;event_type&quot;:&quot;authentication.click&quot;,&quot;payload&quot;:{&quot;location_in_page&quot;:&quot;star button&quot;,&quot;repository_id&quot;:140457254,&quot;auth_type&quot;:&quot;log_in&quot;,&quot;client_id&quot;:null,&quot;originating_request_id&quot;:&quot;83fa:6fe5:e66c1:16b585:5d7e6784&quot;,&quot;originating_url&quot;:&quot;https://github.com/owasp/amass/blob/master/examples/wordlists/alterations.txt&quot;,&quot;referrer&quot;:null,&quot;user_id&quot;:null}}" data-hydro-click-hmac="03e8c55da6ecee2adadd0e3751478ee89795342320061fde36a2352c6e6ff082" href="/login?return_to=%2fowasp%2famass">
```

**Comment by k3mlol** [comment]
Because the release zip file had contained the wordlist files, so if check that 404,I think the program can ignore this error.

**Comment by caffix** [comment]
This has been fixed in v3.1.5
